### DISCUSSION TITLE
Allow PR comments to be pertaining code blocks and not just lines

### DISCUSSION THREAD

**Discussion by ghost** [community/community#36416] [Pull Requests]
Being able to see the surrounding code on a PR comment would be nice. So comments that are about some generalisable trait present in a given section of the code would be easier to understand.

**Edit: So now I just shamefully figured out you can drag the + sign to comment over a block of code.**

**Comment by ossanna16** [comment]
Hey there @adriano-rodrigues-vakt and thank you for the feedback! I'm glad you found our community 🙂

**Comment by queenofcorgis** [comment]
Thanks for updating us (no shame at all) with your discovery - I'm sure you're not the only one who had this thought ‼️

**Reply by zamir-ivry-silk** [reply]
Actually I don't think it is good enough, when I comment about one line of code I want  to be able to see the surrounding area, 
marking the comment as multiline might be a hack, but what I want is to have the comment box not hiding the code but beside it with some mark to indicate on what lines it is commenting.
